src/components/header/Header.tsx

```diff
@@ -1,41 +1,40 @@
 import { useState } from 'react';
 import styled from 'styled-components';
 import { useAuth0 } from '@auth0/auth0-react';
-import { IconButton } from '../general/icon-button/IconButton';
 import { SearchInput } from '../search';
-import { TotalCartQuantityBadge } from '../TotalCartQuantityBadge';
-import { AuthenticationButton } from '../../auth/AuthenticationButton';
 import { CartList } from '../cart-list';
+import { ReactComponent as Cart } from '../../assets/icons/Iconcart.svg';
+import { ReactComponent as Like } from '../../assets/icons/Iconsaved.svg';
+import { ReactComponent as Menu } from '../../assets/icons/Iconmenu.svg';
 
 const HeaderContainer = styled.div`
   display: flex;
   flex-direction: column;
 `;
 
 const StyledHeader = styled.div`
-  background-color: ${({ theme }) => theme.colors.light};
-  padding: ${({ theme }) => theme.sizes.spacing.lg};
+  background-color: ${({ theme }) => theme.colors.Neutral20};
+  padding: ${({ theme }) => theme.sizes.spacing.md};
   display: flex;
   justify-content: space-between;
   align-items: center;
   h1 {
-    font-weight: lighter;
-    font-size: ${({ theme }) => theme.sizes.fonts.xl};
+    font-family: 'Koulen';
```

Comment:
did we say all headers are this font? if so we might be able to set that in GlobalStyle in index.ts